### ISSUE TITLE
Reproducing the paper's results on HMDB51/UCF101

### ISSUE THREAD

**Issue by mengyuest** [TengdaHan/DPC#1]
Hi, I wonder which model (and setting) can we use to reproduce the result for HMDB51/UCF101? I used model [3D-ResNet18-Kinetics400-128x128](https://drive.google.com/file/d/1jbMg2EAX8armIQA6_0YwfATh_h7rQz4u/view?usp=sharing) but only got 1% top-1 accuracy on HMDB51. Thanks for your help.

**Comment by TengdaHan** [comment]
Hi, the weights provided now is the **self-supervised** weights, i.e. the weights haven't seen any action labels. 

To reproduce, you can follow the instruction [here](https://github.com/TengdaHan/DPC#evaluation-supervised-action-classification) with the pretrained weights, and finetune the classifier on UCF101 or HMDB51. 

Email me if you have more questions.

**Comment by TengdaHan** [comment]
Issue has been solved
